GW2EIEvtcParser/EncounterLogic/Fractals/FractalLogic.cs

```diff
@@ -144,6 +145,28 @@ internal override void ComputePlayerCombatReplayActors(AbstractPlayer p, ParsedE
                     }
                 }
             }
+            if (fluxBombSmall != null)
```

Comment:
That's going to put down the same flux bombs for each player. You can put this in a protected static method and call it in ComputeNPCCombatReplay of each logic under the main target of the fight.